baybe/utils/dataframe.py

```diff
@@ -14,12 +14,16 @@
 from baybe.utils.numeric import DTypeFloatNumpy, DTypeFloatTorch
 
 if TYPE_CHECKING:
-    from baybe.campaign import Campaign
     from baybe.parameters import Parameter
+    from baybe.targets.base import Target
 
 # Logging
 _logger = logging.getLogger(__name__)
 
+# TODO: Most functions contained in this module are not utils but strongly coupled
```

Comment:
Did you create a work item for this?

Reply:
No, but we don't have/need work items for all TODOs in the code (there are plenty more without work item). This is rather a minor thing and further refactorings will either take care of it automatically or it will turn out that the TODO is no more relevant at some point. So I'll just keep it here to remind me, but without work item in this case.